samples/sandbox/src/main/kotlin/com/bumble/appyx/sandbox/client/spotlight/SpotlightExampleNode.kt

```diff
@@ -126,78 +139,91 @@ class SpotlightExampleNode(
     @Suppress("LongMethod")
     @Composable
     private fun LoadedState(modifier: Modifier = Modifier) {
         val hasPrevious = spotlight.hasPrevious().collectAsState(initial = false)
         val hasNext = spotlight.hasNext().collectAsState(initial = false)
-        Column(
-            verticalArrangement = Arrangement.spacedBy(24.dp),
-            horizontalAlignment = Alignment.CenterHorizontally,
-            modifier = modifier,
-        ) {
-            Row(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .padding(24.dp),
-                verticalAlignment = Alignment.CenterVertically,
-                horizontalArrangement = Arrangement.SpaceBetween
-            ) {
-                TextButton(
-                    text = "Previous",
-                    enabled = hasPrevious.value
-                ) {
-                    spotlight.previous()
-                }
-                TextButton(
-                    text = "Next",
-                    enabled = hasNext.value
-                ) {
-                    spotlight.next()
-                }
+        val currentTab = spotlight.current().collectAsState(initial = null)
+        Scaffold(
+            modifier = modifier.fillMaxSize(),
+            floatingActionButtonPosition = FabPosition.Center,
+            floatingActionButton = { PageButtons(hasPrevious, hasNext) },
+            bottomBar = {
+                BottomTabs(currentTab)
             }
-            Row(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .padding(24.dp),
-                verticalAlignment = Alignment.CenterVertically,
-                horizontalArrangement = Arrangement.SpaceBetween
-            ) {
-                TextButton(
-                    text = "C1",
-                    enabled = true
-                ) {
-                    spotlight.activate(C1)
-                }
-                TextButton(
-                    text = "C2",
-                    enabled = true
-                ) {
-                    spotlight.activate(C2)
-                }
-                TextButton(
-                    text = "C3",
-                    enabled = true
-                ) {
-                    spotlight.activate(C3)
-                }
-            }
-
+        ) {
             Children(
                 modifier = Modifier
-                    .padding(top = 12.dp, bottom = 12.dp)
-                    .fillMaxWidth(),
-                transitionHandler = rememberSpotlightSlider(clipToBounds = true),
+                    .padding(it),
+                transitionHandler = rememberSpotlightFaderThrough(),
                 navModel = spotlight
             )
+        }
+    }
 
+    @Composable
+    private fun BottomTabs(currentTab: State<NavTarget?>) {
+        NavigationBar {
+            Item.values().forEach {
+                val selected = currentTab.value == it.navTarget
```

Comment:
Give `it` a variable name? The `it`s are used quite a few times and might cause conflicts with other lambas.